.github/workflows/tests.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - uses: ./.github/actions/prepare-environment
 
       - name: Lerna bootstrap
-        run: npx lerna bootstrap --since=$(git merge-base --fork-point origin/dev)
+        run: npx lerna bootstrap --ignore @salutejs/plasma-tokens*
```

Comment:
А почем since не работает ?

Reply:
Так как команда lint запускается не через `lerna` ведет себя оно странновато. Пытается запуститься в пакетах которые не трогали.   

Сделал пока такой вот workaround. (  

Я надеюсь что после того как начнем запускать через lerna то к варианту с `--since ` вернемся.

Reply:
давай это отметим в задаче про lint